projects/demo/src/modules/experimental/navigation/navigation.template.html

```diff
@@ -0,0 +1,67 @@
+<tui-doc-page
+    header="Navigation"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <p>A set of basic layout components for global navigation</p>
+
+        <tui-doc-example
+            id="basic"
+            heading="Basic"
```

Comment:
it seems like the first example is expansive, and the second is basic, isn't?

Reply:
Sure, we can rename them to `Full` and `Expansive header`